README.rst

```diff
@@ -1,6 +1,9 @@
 Toil is a scalable, efficient, cross-platform pipeline management system, written entirely in
 Python, and designed around the principles of functional programming. Full documentation can be found at `Read the Docs`_.
 
+Native system requirements: `python2.7-dev`_, make_, gcc_, `libffi-dev`_,
```

Comment:
This should go into the installation section of the Sphinx doc. And it should be qualified that these are only needed for the encryption extra. I think we already have a section that describes each extra and the native requirements should be listed there.